Distribution/Server/Features/Security/Backup.hs

```diff
@@ -176,6 +176,7 @@ import_v1 = mapM_ fromRecord
     fromInfoRecord [strFileLength, strSHA256] = do
       fileInfoLength <- parseRead "file length" strFileLength
       fileInfoSHA256 <- parseSHA "file SHA256" strSHA256
+      let fileInfoMD5 = Nothing
```

Comment:
So we ought to be exporting the md5 too, and on restore if it's available we should use it. Otherwise this dump/restore will always loose all md5 hashes. There's even a test for this! :-)